packages/universal-components/src/MenuGroup/MenuItem.js

```diff
@@ -34,6 +33,7 @@ export default function MenuItem({
   labelStyle,
   hideActionIcon = false,
 }: Props) {
+  const iconRendered = icon != null;
   const menuIcon =
     icon != null
```

Comment:
I really don't like this, but if I use `iconRendered ? ...` Flow shows errors. Do you have idea how not to repeat the same condition?

Reply:
What about removing `iconRendered` completely. For example like below.
And then just check if any icon will be rendered.

![Screenshot 2019-03-12 at 17 13 05](https://user-images.githubusercontent.com/2660330/54216593-3bedff80-44ea-11e9-8506-63e9506184db.png)

Reply:
Done